frontend/src/components/Board/BuyParts/index.js

```diff
@@ -266,6 +266,9 @@ const RetailerButton = ({
 const StoreIcon = ({ retailer, disabled }) => {
   const imgHref = `/static/images/${retailer}${disabled ? '-grey' : ''}.ico`
   return (
+    // Styling this as a Next Image is unnecessarily complicated, also all next optimizations aren't that useful
```

Comment:
"all next optimizations..." isn't grammatically correct, I suggest "the next optimizations..."

Reply:
Comment is too long now :D

Reply:
Resolved via df88c9b.

Reply:
> Comment line-length is too long now :D